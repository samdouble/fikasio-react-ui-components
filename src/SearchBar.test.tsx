import React from 'react';
import { render } from '@testing-library/react';
import { SearchBar } from '.';

describe('SearchBar', () => {
  it('Renders correctly', () => {
    const { baseElement } = render(<SearchBar />);
    expect(baseElement).toMatchSnapshot();
  });
});
