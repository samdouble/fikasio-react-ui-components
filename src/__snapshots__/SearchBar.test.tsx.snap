// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar Renders correctly 1`] = `
<body>
  <div>
    <div
      class="MuiAutocomplete-root fikasio-searchbar fikasio-theme-dark css-1tlcqt-MuiAutocomplete-root"
      style="outline: none; padding-left: 22px; position: relative; top: -5px;"
    >
      <div
        class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-cmpglg-MuiFormControl-root-MuiTextField-root"
      >
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiAutocomplete-inputRoot css-1blp12k-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <input
            aria-autocomplete="list"
            aria-expanded="false"
            aria-invalid="false"
            autocapitalize="none"
            autocomplete="off"
            class="MuiInputBase-input MuiOutlinedInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused css-16wblaj-MuiInputBase-input-MuiOutlinedInput-input"
            id=":r0:"
            placeholder=""
            role="combobox"
            spellcheck="false"
            type="text"
            value=""
          />
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1ll44ll-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-w4cd9x"
            >
              <span
                aria-hidden="true"
                class="notranslate"
              >
                â€‹
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-magnifying-glass fa-1x "
      data-icon="magnifying-glass"
      data-prefix="fas"
      focusable="false"
      role="img"
      style="bottom: 36px; margin: 5px 5px 5px 10px; position: relative;"
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352a144 144 0 1 0 0-288 144 144 0 1 0 0 288z"
        fill="currentColor"
      />
    </svg>
  </div>
</body>
`;
